services/brig/test/integration/Federation/End2end.hs

```diff
@@ -523,6 +523,11 @@ testSendMessageToRemoteConv brig1 brig2 galley1 galley2 cannon1 = do
           ( OtrMessage bobClient aliceClient (toBase64Text msgText) (Just "")
           )
 
+data NotificationException = NotificationException
+  deriving (Show, Typeable)
+
+instance Exception NotificationException
+
```

Comment:
Is this unused?

Reply:
Yes, I think it was from an attempt to work with Multiverb